CONTRIBUTING.md

```diff
@@ -100,7 +100,9 @@ To be considered popular enough to be within our scope, a brand must be in exist
     - [npm](https://www.npmjs.com): 100k weekly downloads,
     - [jsDelivr](https://www.jsdelivr.com): 1m daily or 35m monthly requests,
     - [crates.io](https://crates.io): 100k weekly downloads,
-    - [PyPi Stats](https://pypistats.org): 100k weekly downloads, or,
+    - [PyPi Stats](https://pypistats.org): 100k weekly downloads,
+    - [Homebrew Formulae](https://formulae.brew.sh): 5k installs in the last 30 days or 50k installs in the last year,
+    - [Arch User Repository](https://aur.archlinux.org): popularity of 7.00 or,
```

Comment:
Can you give some more context to this? What does a popularity of 7.00 mean? What icons do we already have for projects/brands that meet this level of popularity?

Reply:
Quoting the [ArchLinux forum](https://bbs.archlinux.org/viewtopic.php?id=283934):

> Popularity is calculated as the sum of all votes with each vote being weighted with a factor of 0.98 per day since its creation.
>
> Popularity_today = Popularity_yesterday * 0.98 + New_votes_from_today